cli/create.go

```diff
@@ -85,6 +87,12 @@ func createRecordCommand(t *core.Timetrace) *cobra.Command {
 				return
 			}
 
+			now := time.Now()
+			if now.Before(start) || now.Before(end) {
+				out.Err("provided record happends in future!")
```

Comment:
Hello! I'd like to point out a misspelling of `happends` -- should be `happens`.

May I also suggest inserting the article "the" before "future", so that the error message would read as:
`provided record happens in the future!`

Reply:
sure, fixed it 👍